libs/ui/search/src/lib/results-table/results-table.component.html

```diff
@@ -4,16 +4,7 @@
 >
   <!-- SELECTED COLUMN -->
   <gn-ui-interactive-table-column>
-    <ng-template #header>
-      <gn-ui-checkbox
-        [checked]="isAllSelected()"
-        [indeterminate]="isSomeSelected()"
-        (changed)="toggleSelectAll()"
-        type="default"
-        class="-m-2 mr-3"
-      >
-      </gn-ui-checkbox>
-    </ng-template>
+    <ng-template #header> </ng-template>
```

Comment:
Do we need to keep this empty `ng-template` for structure purposes ? or for later?

Reply:
Good point! I think we don't need it anymore.